gtsfm/scene_optimizer.py

```diff
@@ -213,12 +211,20 @@ def create_computation_graph(
         if self._save_bal_files:
             # save the input to Bundle Adjustment (from data association)
             auxiliary_graph_list.append(
-                dask.delayed(write_sfmdata_to_disk)(ba_input_graph, os.path.join(RESULTS_PATH, "ba_input.bal"))
+                dask.delayed(io_utils.write_cameras)(ba_input_graph, save_dir=os.path.join(RESULTS_PATH, "ba_input"))
```

Comment:
@ayushbaid i forgot to ask, does a keyword arg work on dask.delayed?

Reply:
I was not sure too. But i checked and it works.

Reply:
cool